### PR TITLE
chore: remove VerifiedEmail and EmailSent display

### PR DESCRIPTION
**Pull request by nensidosari**
## Changes
- remove VerifiedEmail and EmailSent display

Checked with product here: 
- https://dailydotdev.slack.com/archives/C02AA18RD1D/p1729251570944869
- https://dailydotdev.slack.com/archives/C02AA18RD1D/p1729256868868289

## Events

Did you introduce any new tracking events?

## Experiment

Did you introduce any new experiments?

## Manual Testing

> [!CAUTION]
> Please make sure existing components are not breaking/affected by this PR

### Preview domain
https://chore-cleanup-auth-options.preview.app.daily.dev

### CODE REVIEW

File: packages/shared/src/components/auth/AuthOptions.tsx [hunk 1]
```diff
@@ -94,28 +92,13 @@ const AuthHeader = dynamic(
   () => import(/* webpackChunkName: "authHeader" */ './AuthHeader'),
 );
 
-const EmailVerificationSent = dynamic(
-  () =>
-    import(
-      /* webpackChunkName: "emailVerificationSent" */ './EmailVerificationSent'
-    ),
-);
```
Comment (capJavert) [inline packages/shared/src/components/auth/AuthOptions.tsx:102]: We just don't use it?

Comment (nensidosari) [reply packages/shared/src/components/auth/AuthOptions.tsx:102]: Exactly, old code never rendering. Checked here also:

- https://dailydotdev.slack.com/archives/C02AA18RD1D/p1729251570944869
- https://dailydotdev.slack.com/archives/C02AA18RD1D/p1729256868868289

File: packages/shared/src/components/auth/AuthOptions.tsx [hunk 2]
```diff
@@ -94,28 +92,13 @@ const AuthHeader = dynamic(
...
 );
 
-const EmailVerified = dynamic(
-  () => import(/* webpackChunkName: "emailVerified" */ './EmailVerified'),
-);
-
-const LoginForm = dynamic(
-  () => import(/* webpackChunkName: "loginForm" */ './LoginForm'),
-);
-
```
[the remark this answers is not in the document]
Comment (nensidosari) [reply packages/shared/src/components/auth/AuthOptions.tsx:118]: This was removed from imports here as it's not used in AuthOptions, but the form itself its being used elsewhere so not something we can cleanup

**Comment by nensidosari** [comment]
>  would suggest we do critical flow test on prod deployment?

@rebelchris yup of course 🙌 I'll make sure I do once deployed, I've done on preview already

**Pull request merged**